notebooks/writing_a_trace_transform_cpu_offloading.ipynb

```diff
@@ -0,0 +1,873 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #147.                if vout := variableify(out) in offloaded_tensors:
</pre><p>quite nitpicky and cosmetic but how about?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Lightning-AI/lightning-thunder/pull/966/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>